test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpTraceExporterTests.cs

```diff
@@ -348,6 +348,97 @@ public void ToOtlpSpanActivitiesWithNullArrayTest()
             Assert.Null(stringArray[2].Value);
         }
 
+        [Theory]
+        [InlineData(ActivityStatusCode.Unset, "")]
+        [InlineData(ActivityStatusCode.Ok, "")]
+        [InlineData(ActivityStatusCode.Error, "error description")]
+        public void ToOtlpSpanNativeActivityStatusTest(ActivityStatusCode expectedStatusCode, string statusDescription)
+        {
+            using var activitySource = new ActivitySource(nameof(this.ToOtlpSpanTest));
+            using var activity = activitySource.StartActivity("Name");
+            activity.SetStatus(expectedStatusCode, statusDescription);
+
+            var otlpSpan = activity.ToOtlpSpan();
+
+            if (expectedStatusCode == ActivityStatusCode.Unset)
+            {
+                Assert.Null(otlpSpan.Status);
+            }
+            else
+            {
+                Assert.NotNull(otlpSpan.Status);
+                Assert.Equal((int)expectedStatusCode, (int)otlpSpan.Status.Code);
+                if (expectedStatusCode == ActivityStatusCode.Error)
+                {
+                    Assert.Equal(statusDescription, otlpSpan.Status.Message);
+                }
+
+                if (expectedStatusCode == ActivityStatusCode.Ok)
+                {
+                    Assert.Empty(otlpSpan.Status.Message);
+                }
+            }
+        }
+
+        [Theory]
+        [InlineData(StatusCode.Unset, "unset", "")]
+        [InlineData(StatusCode.Ok, "Ok", "")]
+        [InlineData(StatusCode.Error, "ERROR", "error description")]
```

Comment:
```suggestion
        [InlineData(StatusCode.Unset, "Unset", "")]
        [InlineData(StatusCode.Ok, "Ok", "")]
        [InlineData(StatusCode.Error, "Error", "error description")]
```

Reply:
What's the best practice for status description? Is it preferred to have the first letter uppercase? Is it preferred to have the ending `.`?

Reply:
I don't think think the exporter should alter the description provided. Or are you saying the unit tests should model a nicely formatted description?

Reply:
I didn't see Description format being specified in the spec. But I think using a sentence to describe it makes more sense for a error description.

Regarding the suggested change, I wanted to exercise this code path:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/98cbd56ca26455046f953ce2d1aaf42968840fa7/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs#L260
by using "unset" and "ERROR" as the statusCodeTagValue. I can add these as separate testcases if that is clearer.

Reply:
I feel the unit test wasn't clear about what's the intention of these different uppercase/lowercase/wording. Maybe these were intentional?

Reply:
BTW, no need to treat this as a blocker.

Reply:
I've added a dedicated testcase for statusCodeTagValue case sensitivity.